lib/game.js

```diff
@@ -0,0 +1,64 @@
+const Frame = require('./frame');
+
+class Game {
+  constructor () {
+    this.rollHistory = [];
+    this.currentFrame = 0;
+    this.totalScore = 0;
+    this.frames = [
```

Comment:
This seems like a much better idea than what I did, It's clean and simple there will never be more or less than 10 frames in a game